frontend/src/utils/StudentUtils.ts

```diff
@@ -76,3 +76,13 @@ export const questionStatus = (
   if (answers[index].isCompleted) return QuestionNumberTypes.COMPLETED;
   return QuestionNumberTypes.UNATTEMPTED;
 };
+
+export const mapAnswersToResultsArray = (answers: Answers[]): number[][][] => {
+  return answers.map((answer) => {
+    const row: number[][] = [];
+    answer.elements.map((element) => {
```

Comment:
I think you should be able to do:

```suggestion
    answer.elements.forEach((element) => {
```
or replace the whole thing with:
```tsx
  return answers.map((answer) => answer.elements.map((element) => element.elementAnswers))
```

Reply:
nice, thank you!